addons/src/createAutoCorrectedDatePipe.js

```diff
@@ -1,9 +1,13 @@
-export default function createAutoCorrectedDatePipe(dateFormat = 'mm dd yyyy') {
+export default function createAutoCorrectedDatePipe({
+  dateFormat = 'mm dd yyyy',
+  minYear = 1,
+  maxYear = 9999
+} = {}) {
```

Comment:
@dclark27 For backwards compatibility, can we keep the first argument as it was, and make the options param a second argument.

```
export default function createAutoCorrectedDatePipe(dateFormat = 'mm dd yyyy', {
  minYear = 1,
  maxYear = 9999
} = {}) {
```